hudi-common/src/main/avro/HoodieMetadata.avsc

```diff
@@ -121,16 +121,192 @@
                             "doc": "Minimum value in the range. Based on user data table schema, we can convert this to appropriate type",
                             "name": "minValue",
                             "type": [
+                                // Those types should be aligned with Parquet `Statistics` impl
+                                // making sure that we implement semantic consistent across file formats
+                                //
+                                // NOTE: Other logical types (decimal, date, timestamp, etc) will be converted
+                                //       into one of the following types, making sure that their corresponding
+                                //       ordering is preserved
                                 "null",
-                                "string"
+                                {
```

Comment:
should we keep `"string"` for compatibility across SNAPSHOTS releases?

Reply:
Not sure i follow your point

Reply:
Per discussion, this is not a problem for public releases, so it's okay to remove it.